src/MongoId.php

```diff
@@ -178,13 +181,15 @@ public function getInc()
      *   string consisting of exactly 24 hexadecimal characters; otherwise,
      *   FALSE is returned.
      */
-    public static function isValid($id)
+    public static function isValid($value)
     {
-        if (!is_string($id)) {
+        if (get_class($value) == self::class) {
```

Comment:
That is an valid notation.
Please check Symfony 2.8 Forms, we can provide now an Object::class definition